jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -0,0 +1,364 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Deploy to Amazon SageMaker
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A short page description goes here
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available. The link:https://circleci.com/developer/orbs/orb/circleci/aws-sagemaker[SageMaker orb] is also in **open preview**.
+
+This how-to guide walks you through the steps to set up and use the Amazon SageMaker orb to deploy and release models to SageMaker. You will track releases in the CircleCI release dashboard. This gives you the ability to manage promotions and rollbacks. You will orchestrate model deployments to endpoints across different environments. The link:https://github.com/CircleCI-Public/sagemaker-deploy-examples#[example project repository] allows you to train a new model package version and then deploy it across multiple environments.
```

Comment:
Question - are we calling it Release Dashboard? Or Releases UI? 
<img width="909" alt="Screenshot 2023-11-27 at 14 22 25" src="https://github.com/circleci/circleci-docs/assets/19594309/a52588d0-010b-4f48-86d1-3da4c160a3d2">

I'd used Releases UI link in the Changelog, but I think it's better to keep consistent. If Release Dashboard is the official name, we should use this also in the blog article and all other comms

Reply:
> You will track releases in the CircleCI release dashboard. This gives you the ability to manage promotions and rollbacks. You will orchestrate model deployments to endpoints across different environments.
I would drop `This gives you the ability to manage promotions and rollbacks.`  because this is actually only available for CircleCI projects so far, so I don't think we should call it out. Using a generic "manage" I think works. 
I also would move the part about "orchestrating" _before_ we talk about the releases UI, just because in theory you can do the orchestration without integrating with the releases UI. 
> You will orchestrate model deployment to endpoints across different environments. You will also be able to track and manage your releases in the CircleCI release dashboard.

And a tiny thing: 
> ... allows you to train a new 
I would change to
> ...*.*shows you how to** train .... 
because the training part is really not something that we help much on, we are just trying to onboarding people through the repo and get them started fast

Reply:
I think we use release dashboard to mean just the page you get to when you select releases in the sidebar. I'm fine with using releases UI to mean all the different pages you can access in the releases section. I'll update this doc to be more generic - use releases UI. Also yes to your other suggestions